loctool.js

```diff
@@ -83,6 +83,10 @@ function usage() {
         "  Specify the dir where the xliffs files live. Default: \".\"\n" +
         "--localizeOnly\n" +
         "  Generate a localization resource only. Do not create any other files at all after running loctool. \n" +
+        "--xliffResRoot\n" +
+        "  Specify the dir where the xliffs resource generation output (Default is resources/) live.. \n" +
```

Comment:
Make this `Specify the dir where the generation output should go. (Default is resources/)`

Reply:
Update it. Thanks!